src/main/java/org/fundacion/jala/converter/controller/AuthController.java

```diff
@@ -1,10 +1,12 @@
 /**
  * Copyright (c) 2021 Fundacion Jala.
- * <p>
+ *
  * This software is the confidential and proprietary information of Fundacion Jala
  * ("Confidential Information"). You shall not disclose such Confidential
  * Information and shall use it only in accordance with the terms of the
  * license agreement you entered into with Fundacion Jala
+ *
+ * @author Raymundo Guaraguara Sansusty
  */
 package org.fundacion.jala.converter.controller;
```

Comment:
A space is missing between line 10 and 11

Reply:
We agreed there would be no space between copyright and packages.